src/test/java/nl/tudelft/ewi/devhub/modules/MockedGitoliteGitServerModule.java

```diff
@@ -126,7 +126,6 @@ protected void createMockedMirrorsFolder() {
 	protected void createMockedRepositoriesFolder() {
 		String repositoriesPath = repositoriesFolder.toPath().toString() + "/";
 		when(configuration.getGitoliteBaseUrl()).thenReturn(repositoriesPath);
-		when(configuration.getRepositoriesDirectory()).thenReturn(repositoriesFolder);
```

Comment:
As far as I remember, this stubbing is required for testing repository removal. It may be unused now, but this stubbing is required for the mocked git server module to work as expected as far as I remember. I remember it being used in the version for the `git-server` tests (of which this one is a fork).